Makefile

```diff
@@ -454,6 +457,8 @@ build-docker-e2e:
 	$(MAKE) -C contrib/images babylond-e2e
 	$(MAKE) -C contrib/images babylond-before-upgrade
 	$(MAKE) -C contrib/images e2e-init-chain
+	$(MAKE) -C test/e2e/bcd_integration build-integration-test
```

Comment:
these are now in contrib/images https://github.com/babylonlabs-io/babylon/pull/57/files#diff-b7a8cc1d7346184c68e119aba8c13a202872c6deccf916e6e2b5216b4a6778ceR49-R55